test/jdk/jdk/nio/zipfs/TestPosix.java

```diff
@@ -219,34 +215,27 @@ static interface Executor {
 
     private static String expectedDefaultOwner(Path zf) {
         try {
-            try {
-                PrivilegedExceptionAction<String> pa = ()->Files.getOwner(zf).getName();
-                return AccessController.doPrivileged(pa);
-            } catch (UnsupportedOperationException e) {
-                // if we can't get the owner of the file, we fall back to system property user.name
-                PrivilegedAction<String> pa = ()->System.getProperty("user.name");
-                return AccessController.doPrivileged(pa);
-            }
-        } catch (PrivilegedActionException | SecurityException e) {
+            return Files.getOwner(zf).getName();
+        } catch (UnsupportedOperationException | NoSuchFileException e) {
+            // if we can't get the owner of the file, we fall back to system property user.name
+            return System.getProperty("user.name");
+        } catch (IOException e) {
             System.out.println("Caught " + e.getClass().getName() + "(" + e.getMessage() +
-                ") when running a privileged operation to get the default owner.");
+                    ") when running a privileged operation to get the default owner.");
```

Comment:
Given we are not running a privileged operation, we should probably tweak the exception message

Reply:
Thanks, I was a bit of a robot here and didn't read the exception message :-)

I think it might be better to simply propagate this IOException to the calling method `testPosixDefaults`. This method throws IOException. Catching this IOException here provides limited value.

What do you think?